src/meltano/cli/schedule.py

```diff
@@ -207,7 +207,7 @@ def _format_elt_list_output(entry: Schedule, session: Session) -> dict:
     last_successful_run = entry.last_successful_run(session)
     last_successful_run_ended_at = (
         last_successful_run.ended_at.isoformat()
-        if last_successful_run.ended_at
+        if last_successful_run and last_successful_run.ended_at
```

Comment:
This actually fixes a bug that occurs when running `meltano schedule list --format json` when the `runs` table in the System DB is empty (i.e. a fresh new project):

```shell
$ meltano init --no-usage-stats .
...

$ meltano add extractor tap-readthedocs --no-install
...

$ meltano add loader target-jsonl
...

$ meltano schedule add --extractor tap-readthedocs --loader target-jsonl test --interval='@once'
...

$ meltano schedule list --format json
2024-11-07T04:14:43.512502Z [info     ] The default environment 'dev' will be ignored for `meltano schedule`. To configure a specific environment, please use the option `--environment=<environment name>`.
Need help fixing this problem? Visit http://melta.no/ for troubleshooting steps, or to
join our friendly Slack community.

'NoneType' object has no attribute 'ended_at'
```